rental.rb

```diff
@@ -0,0 +1,12 @@
+# Defines the Rental class blueprint
+class Rental
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    person.rentals << self
+  end
+
+  attr_accessor :date, :person, :book
+end
```

Comment:
- [ ] You can make your Class Rental look nicer as follow and remove the instance variable "@rentals" in the method "add_rental in the class" Person"".

```ruby
class Rental
  attr_accessor :date, :book, :person

  def initialize(date, book, person)
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
  end
end
```